### ISSUE TITLE
FO4 crashing after creating save on Win 11

### ISSUE THREAD

**Issue by tr0ubadour** [ModOrganizer2/modorganizer#1596]
## The problem:
Everytime I save, the game produces a successful save (which I can continue from after launching the game again) , but crashes immediately afterwards. Each Buffout4 crash log shows the usvfs_x64.dll at the top of the "STACK" section.
Fudgyduff, the creator of Buffout 4 recommended to post this issue here.
I've already tested with the "usvfs-insider-v2"-files from the pinned issue on Win 10 with the same result.

## To Reproduce:
Just create an autosave. Creating a normal save using the console crashes the game, but doesn't create a buffout crashlog, so this couldn't be properly tested.

## Environment:

* Mod Organizer Version that exhibits the issue: v2.4.2
* Last Mod Organizer Version that did not exhibit the issue (if applicable): -
* Desktop OS/version used to run Mod Organizer: Windows 11 Pro Build 22000.194

## Details:
Buffout 4 crashlog: https://gist.github.com/tr0ubadour/5822e94a1eaa779dc649c03738bb18ed

## Link to Mod Organizer logs:
Right click on the log window at the bottom of MO2, set log level to debug, restart Mo2 and reproduce the problem.
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

### USVFS:
https://gist.github.com/tr0ubadour/4a149a27b2c462d2dcf98152bf7afd30

### MO Interface:
https://gist.github.com/tr0ubadour/a04bd8cdd0ac0a34d7dbe8dec2c0040c

**Comment by tr0ubadour** [comment]
Little update: The crash seems to be caused by the game deleting any save.
On a fresh game the first saves work without a crash, but as soon as the game deletes an older autosave, or you manually delete any save, the CTD happens.

**Comment by LostDragonist** [comment]
I haven't been able to reproduce this yet though I'm on a completely vanilla install right now.  Any chance you can try a new profile in MO with no mods and see if the issue persists there?

**Comment by tr0ubadour** [comment]
I did as you requested and started a new game with a new profile with no mods activated in MO.
The game crashed after creating the third autosave after the character creation screen and also after starting the game again, when I tried deleting a save in the main menu.

Here are the logs:

##USVFS:
https://gist.github.com/tr0ubadour/68fcf1d477693489f6c10087df9fd1e5

##MO Interface:
https://gist.github.com/tr0ubadour/dca38c7451a17e1d438c56e8985a77dd

**Comment by LostDragonist** [comment]
Yes, I tested on Windows 11. I spent hours installing it just for you.

> 20:19:30.327 [I] hooked NtClose (0x7fffec6034a0) in C:\WINDOWS\SYSTEM32\ntdll.dll type chained patch
> 20:19:30.327 [I] existing hook to 7fff6c651100 in unknown

This indicates you have some sort of anti-virus or anti-virus like software installed that isn't registered with Windows.  I've seen this before with a program called Seguarzo that is really malware disguised as an anti-virus.  I'm guessing you have something like that going which is causing problems.

**Comment by Betaglutamate** [comment]
Hi,

I had this exact issue managed to solve via bitdefender exclusions. (You can add exclusions in the bitdefender program)

I added the following paths:

Excluded:
MO2 folder
modorganizer.exe,
SSE folder
SKSE64_loader.exe
skyrimse.exe

This solved the problem

**Comment by arturek1666** [comment]
I know this is old thread however the issue still shows in 2022, and indeed the fix is to add exception in Bitdefender, however for safety I would recommend only adding SkyrimSE.exe to exception that is enough as tested. 

It's mostly Advanced Threat Defence and you just add exception there as main file of  the game with is: SkyrimSE.exe nothing else non of those required to add:
MO2 folder
modorganizer.exe,
SSE folder
SKSE64_loader.exe

**Comment by Johnnyx92** [comment]
Can confirm and it's same for Skyrim AE, after two days of constant reinstalling MO2, fresh installing Skyrim AE over and over with SKSE.

Turns out it was Bitdefender ¤%#&!. Even worse is it was advanced threat defenderthat was the issue because I earlier tried only deactivating anti virus (not advanced) and still didn't work.

I whitelisted everything Betaglutamate mentioned and extra. But as Arturek1666 said it should be enough with Skyrim SE.exe add as exception.

**Comment by mvinc006** [comment]
Appreciate people reporting this. Spent two days dealing with crashes due to bit defender and how it requires you to manually add the. Exe to the exceptions, it's baffling

**Comment by Dinerrty** [comment]
Thank you so much for this, it's been baffling me!, I can't believe it came down to Bitdefender!, I just added an exception to Skyrim SE.exe and I can save fine through MO2 with SKSE installed. Thanks again.
